### DISCUSSION TITLE
Algum cargo pode alterar o saldo do cliente? Se sim, quais?

### DISCUSSION THREAD

**Discussion by lucassl2020** [EuricoDNJR/SG-VILLA-DOLCE#73] [Q&A]
Por enquanto, existem 2 cargos no sistema. O gerente e o funcionário. Algum desses cargos possuem a autorização para alterar o saldo do cliente?

**Comment by EuricoDNJR** [comment]
No momento, nenhum dos dois cargos. Queria discutir com vocês sobre tal atributo ser alterável de maneira "direta" apenas pelo gerente, por exemplo, ele indo em **Clientes->Selecionar um Cliente->Ver->Ação: "Alterar o saldo manualmente"**. E ele ser alterável por parte do funcionário, e obviamente o gerente, de forma indireta, ou seja, na realização de um pedido.

**Reply by lucassl2020** [reply]
Discutir sobre a possibilidade de adicionar uma opção (permitida apenas para o gerente) de tornar o saldo editável para ambos os cargos?
